src/Components/Blueprints/BlueprintsPagination.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import {
+  OnSetPage,
+  Pagination,
+  PaginationVariant,
+} from '@patternfly/react-core';
+
+import {
+  selectBlueprintSearchInput,
+  selectLimit,
+  selectOffset,
+  setBlueprintLimit,
+  setBlueprintsOffset,
+} from '../../store/BlueprintSlice';
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import { useGetBlueprintsQuery } from '../../store/imageBuilderApi';
+
+const BlueprintsPagination = () => {
+  const blueprintSearchInput = useAppSelector(selectBlueprintSearchInput);
+  const blueprintsOffset = useAppSelector(selectOffset);
+  const blueprintsLimit = useAppSelector(selectLimit);
+  const currPage =
+    Math.floor((blueprintsOffset || 0) / (blueprintsLimit || 0)) + 1;
```

Comment:
I'd rather default to something that works :P (ideally defaultLimit if we can get it, if not, some reasonable value):
```suggestion
    Math.floor((blueprintsOffset || 0) / (blueprintsLimit || 10)) + 1;
```

Reply:
TBH I think that expecting there will always be a value is fair too, we do have default in the Slice https://github.com/osbuild/image-builder-frontend/blob/b5c3f12db2139c1d8f32ab1818fb5226b14672b2/src/store/BlueprintSlice.ts#L19